beyond-essence/help-center/design-and-formatting.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: beyond-essence
+header_image: private/main.jpg
+title: Design & formatting
+background_image: index.jpg
+---
+
+The design and the formatting in our Help center differs in some points from the design and style of our software.
+
+## Keep the formatting structured and simple
+
```

Comment:
Tried to play around a bit for the blueprint. It was pretty hard to come up with a blueprint that only had "This is..."-text because the context for menu items and buttons was weird. That's why I included a screenshot which shows every formatting option in a real article. I hope that also helps.